docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -259,6 +259,12 @@ V&V Checks:
     - At home (10%), in public/governmental hospital (2%), in private for-profit health facility (22%), and private not-for-profit health facility (4%), country-specific governmental health facility (10%)  
     - EmONC (Ethiopia; Table 10.5.4A)
     - These are placeholder values and will be updated with further analysis. We want these to be location specific, please code accordingly. The denominator for these values is LBW and preterm births. Outstanding items: believe this only affected neonatal outcomes, confirm with BMGF
+  * - 8
+    - % of those who receive intrapartum sensors that get identified as needing a c-section
```

Comment:
I think it might be hard to find an evidence-based number to use here --- have you seen this sort of quantity in the literature yet?  Let's make it high-priority to confirm that we will be able to find something for tihs.

Reply:
Intrapartum sensors don't even exist yet, right? What sort of data would we look for that would be relevant to this? Also, we still haven't figured out in general how to identify which simulants need a C-section.

Reply:
Yes, as I understand, this would be more of a hypothesized data point based on what Gates can tell us!